src/com/sfuentes/AverageTest.java

```diff
@@ -3,35 +3,42 @@
 public class AverageTest {
 
   public static void main(String[] args) {
-    validateAverageLessThanTen();
-    validateAverageIsDecimal();
+    calculateAverageStudent_averageIsCorrect();
+    calculateAverageStudent_decimals();
 
   }
 
-  public static void validateAverageLessThanTen() {
-    Average less = new Average();
-    double result = less.calculateAverageStudent();
-    System.out.println("Test case 1 validateAverageLessThanTen: \n\t" +
-        "Expected result: range 0 - 10 \n\t" +
+  public static void calculateAverageStudent_averageIsCorrect() {
+    Average correct = new Average();
+    double[] grades = {8,8,8,8,8,8,8};
+    double result = correct.calculateAverageStudent(grades);
+    System.out.println("Test case 1 Calculated average is correct: \n\t" +
+        "Expected result: 8 \n\t" +
         "Obtained result: " + result);
-    if (result <= 10) {
+    if (result == 8) {
       System.out.println("\tSuccessful test.");
     } else {
       System.out.println("\tTest failed.");
     }
   }
 
-  public static void validateAverageIsDecimal() {
+  public static void calculateAverageStudent_decimals() {
     Average decimal = new Average();
-    double result = decimal.calculateAverageStudent();
-    System.out.println("Test case 1 validateAverageLessThanTen: \n\t" +
-        "Expected result: range 0.0 -10.0 \n\t" +
+    double[] grades = {5.5,9.1,7.5,8,8,8,8.3};
+    double result = decimal.calculateAverageStudent(grades);
+    result = Math.round(result*100.0)/100.0;
```

Comment:
What is the purpose of this line?

Reply:
line 27: I'm declaring my array with your values
line 28: I'm declaring a result variable and I'm assigning an object and my method calculateAverageStudent.
line 29: I'm limiting the decimals to 2 so that when validating I can compare the expected result with the result of the method to be tested

Reply:
I have already modified the whole method.